src/Home/tile.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Center, Heading, Pressable } from 'native-base';
+import React from 'react';
+
+interface PropsTitle {
+  bgLinearColors: string[];
+  heading: string;
+  body: any;
+  onPress?: () => void;
+}
+export function Tile({ bgLinearColors, heading, body, onPress }: PropsTitle) {
```

Comment:
In my opinion, it's better to name component files with a capital letter: `TItle.jsx` instead of `title.jsx`.